src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -1815,6 +2348,199 @@ public static partial class Kernel32
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern unsafe bool SetInformationJobObject(SafeObjectHandle hJob, JOBOBJECT_INFO_CLASS jobObjectInfoClass, void* lpJobObjectInfo, uint cbJobObjectInfoLength);
 
+        [DllImport(nameof(Kernel32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool GenerateConsoleCtrlEvent(uint dwCtrlEvent, uint dwProcessGroupId);
+
+        [DllImport(nameof(Kernel32), CharSet = CharSet.Unicode, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool AddConsoleAlias(string Source, string Target, string ExeName);
+
+        [DllImport(nameof(Kernel32), SetLastError = true)]
+        public static extern IntPtr CreateConsoleScreenBuffer(uint dwDesiredAccess, uint dwShareMode, ref SECURITY_ATTRIBUTES lpSecurityAttributes, uint dwFlags, IntPtr lpScreenBufferData);
+
+        [DllImport(nameof(Kernel32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool AttachConsole(uint dwProcessId);
+
+        [DllImport(nameof(Kernel32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool FlushConsoleInputBuffer(IntPtr hConsoleInput);
+
+        [DllImport(nameof(Kernel32), CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern unsafe uint GetConsoleAliases(
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Array)] char* lpAliasBuffer,
+            int AliasBufferLength,
+            string lpExeName);
+
+        [DllImport(nameof(Kernel32), CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern uint GetConsoleAliasesLength(string lpExeName);
+
+        [DllImport(nameof(Kernel32), CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern unsafe uint GetConsoleAliasExes(
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Array)] char* lpExeNameBuffer,
+            uint ExeNameBufferLength);
```

Comment:
`int` instead of `uint` for the second parameter type, since it's a length that will most likely be provided from a caller that's using `int` (calling `Array.Length` to get the value).
You do this in other methods. Just missed this one, I guess.